osu.Game.Rulesets.Osu/UI/OsuPlayfieldAdjustmentContainer.cs

```diff
@@ -55,6 +55,7 @@ protected override void Update()
                 // Scale = 819.2 / 512
                 // Scale = 1.6
                 Scale = new Vector2(Parent.ChildSize.X / OsuPlayfield.BASE_SIZE.X);
+                Position = new Vector2(0, 8 * Parent.ChildSize.Y / OsuPlayfield.BASE_SIZE.Y);
```

Comment:
This should probably have a disable for the editor, as stable does. Exposing it as a property seems to be the easiest way to do so and then setting it in `DrawableOsuEditRuleset`.

Reply:
I completely forgot that this also affects the editor.
Do you mean as a `PlayfieldShift` property with `8` set as default which is then set to 0 by `DrawableOsuEditRuleset`?

Reply:
Yeah, that would be fine.

Reply:
for future proofing, how would this work with the design mode in editor?

Reply:
Does design mode turn this off somehow? Or is the exclusion for the editor just because of the menu bar (in which case this probably shouldn't exist)?

Reply:
as in we want the storyboard to align correctly in design mode too. so i'm not sure it should ever be turned off?

Reply:
Yeah that makes sense. Let's go with this as is then.

Reply:
So maybe better solution is to use bool `PlayfieldShift` as a property? It will be better than passing arbitrary values to the object. I think we want to turn on shift in design mode because aligning storyboard objects to hitobjects in osu-stable editor is a huge pain.

Reply:
If we're looking to fix historically awkward offsets, this could potentially be applied at the legacy storyboard parsing step (moving all elements to their correct location) rather than adding local offsets.

I'd ask that we don't move too quickly on merging this and actually get it right.

Reply:
That was brought up in the original thread, with the issue that it creates a black border at the bottom of the screen.

Reply:
Am I missing something? stable applies this offset to gameplay too. So if we apply the same offset here, and change nothing about the editor and keep the offset, then things will align correctly and the saved positions would be post-offset so that gameplay should still match up when imported into stable?

Reply:
if we are comfortable with keeping the offset as-is going forward then yes, applying at the playfield is the correct solution. given it's something like 12 "gameplay" pixels i'm not sure we want to, but maybe more effort to break compatibility than keep it at this point?